src/form_helper.js

```diff
@@ -256,36 +245,41 @@ export default class FormHelper {
   }
 
   _setElementValue(element, value, elementName){
-    if (element) {
-      element.value = value;
+    if (!element) {
+      this.logErrorMessage(element, value, elementName)
```

Comment:
hmmm - does this work? the `logErrorMessage` function appears to take `0` parameters.